light/client.go

```diff
@@ -368,11 +368,6 @@ func (c *Client) initializeWithTrustOptions(options TrustOptions) error {
 		return fmt.Errorf("expected header's hash %X, but got %X", options.Hash, h.Hash())
 	}
 
-	err = c.compareNewHeaderWithWitnesses(h)
```

Comment:
also don't forget to update the comment at the top of the function since we are no longer comparing with witnesses (btw I also agree that this is probably redundant)